src/solvers/refinement/string_constraint_generator_valueof.cpp

```diff
@@ -420,7 +420,9 @@ exprt string_constraint_generatort::add_axioms_for_parse_int(
   PRECONDITION(f.arguments().size()==1 || f.arguments().size()==2);
   string_exprt str=get_string_expr(f.arguments()[0]);
   const exprt radix=
-    f.arguments().size()==1?from_integer(10, f.type()):f.arguments()[1];
+    f.arguments().size()==1?
+      static_cast<exprt>(from_integer(10, f.type())):
```

Comment:
The compiler complained if the two options in the ternary expression weren't the same type